Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -651,6 +651,10 @@
 #define ENABLE_MODEL_ELEMENT_STAGE_MODE_INTERACTION ENABLE_MODEL_PROCESS
 #endif
 
+#if !defined(ENABLE_SCENE_GEOMETRY_UPDATE) && PLATFORM(VISION) && USE(APPLE_INTERNAL_SDK)
```

Comment:
If not in this PR, can you file a followup radar to remove `USE(APPLE_INTERNAL_SDK)` as soon as possible?